test/agent-policy-tests/test-runner.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
Instead of making a separate script for this, you can [add additional tests](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/build/developer-tools/build/scripts/task_helper_functions.sh#L514) to the helper functions ([for example](https://github.com/terraform-google-modules/terraform-google-startup-scripts/blob/master/test/task_helper_functions.sh)).

Reply:
+1; I ended up writing a custom function + adding a target in the makefile that runs the function with the handwritten integration tests. For the unit tests, I have another target in the makefile that runs the unit tests (now refactored to use bats).